### ISSUE TITLE
youtube-control appears and disappears 

### ISSUE THREAD

**Issue by scmarcus** [shirohana/alfred3-youtube-control#3]
Hi, 
when I open Alfred and type y, youtube control appears and disappears immediately. I can not choose it. I only see the icon appearing but not more.

I'm using the latest Alfred version and also the latest macOS version.

**Comment by shirohana** [comment]
Hello ٩(｡・ω・｡)

Thank you for reporting, it seems like a bug (´・ω・｀)

It could caused by many different way:

- Google Chrome changed the supports of AppleScript (accessibility of AppleScript)
- YouTube changed the DOM structure of YouTube Player
- After OS X was upgraded, Alfred may re-asks for system permissions
- Apple changed their API of AppleScript

so I need you to help me to ensure the problem. Please tell me that:

- Is it you're using the latest version of Google Chrome?
- The full link of the YouTube page
- Have you changed any settings on your [YouTube Account Page](https://www.youtube.com/account)?
- Have you installed any extensions that about YouTube, or browser appearance/behavior on Google Chrome?
- We detect the pages which ends with `- YouTube` as a YouTube Player. Has your YouTube tab contains `- YouTube` at the end during playing/pause?
- Have you tried to restart Google Chrome?
- Can you control your Player by setting keys like:

![image](https://user-images.githubusercontent.com/11359892/52882117-a472e680-31a1-11e9-9624-2d205f9e7935.png)

Thanks first ฅ' ω 'ฅ

**Comment by scmarcus** [comment]
* Is it you're using the latest version of Google Chrome?
yes
* The full link of the YouTube page
https://www.youtube.com/watch?v=iUohO2MSot8
* Have you changed any settings on your [YouTube Account Page](https://www.youtube.com/account)?
no
* Have you installed any extensions that about YouTube, or browser appearance/behavior on Google Chrome?
no I'm using a clean chrome browser, because I downloaded it only for this purpose.
* We detect the pages which ends with `- YouTube` as a YouTube Player. Has your YouTube tab contains `- YouTube` at the end during playing/pause?

* Have you tried to restart Google Chrome?
yes
* Can you control your Player by setting keys like:
no

**Comment by shirohana** [comment]
Is there any message if you enabled debug mode?

![2019-02-18 11 41 30](https://user-images.githubusercontent.com/11359892/52961932-01a3ad80-33d7-11e9-9017-2cb20ec5fc7a.png)

**Comment by scmarcus** [comment]
[2019-02-19 12:34:47][ERROR: input.scriptfilter] Code 1: 0:23: execution error: Google Chrome got an error: Executing JavaScript through AppleScript is turned off. To turn it on, from the menu bar, go to View > Developer > Allow JavaScript from Apple Events. For more information: https://support.google.com/chrome/?p=applescript (12)

This was the error message I got. I switched it on and now it works.
sometimes it is that easy...
Thanks for your help
